public/controllers/LoginController.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Descriere:
+ * Author: 
+ */
+
+
+var app = angular.module('contacts');
+app.controller('LoginController', function($http,$state){
+    var vm = this;
+
+     vm.user = {};
+
+    vm.login = function() {
+
+
+       $http({
+            method: 'POST',
+            url: '/v1/session',
+            data: vm.user
+
+        }).then(function (response){
+            
+        });
+
+  $state.go('dashboard');
```

Comment:
Redirectul asta nu e ok. L-ai pus aici si o sa faca redirect fix cand se initializeaza pagina. Redirect-ul asta ar trebui sa fie la linia 22, in partea de "then", adica dupa ce vine raspunsul de la server